zebrad/src/components/mempool/downloads.rs

```diff
@@ -65,6 +64,7 @@ pub(crate) const TRANSACTION_VERIFY_TIMEOUT: Duration = BLOCK_VERIFY_TIMEOUT;
 const MAX_INBOUND_CONCURRENCY: usize = 10;
 
 /// The action taken in response to a peer's gossiped transaction hash.
+#[derive(Debug)]
 pub enum DownloadAction {
```

Comment:
Since this is `downloads::DownloadAction`, maybe it should just be `downloads::Action`?

Reply:
I agree, but ended up being removed in 8c29d13